tutoraspects/patches/k8s-deployments

```diff
@@ -216,6 +216,8 @@ spec:
               name: docker
             - mountPath: /app/pythonpath
               name: pythonpath
+            - mountPath: /app/pythonpath/openedx
```

Comment:
Just out of curiosity, why were these moves necessary?

Reply:
They upgraded the python version and that causes issues with imports without a module